### PR TITLE
Fix typos in data_adapter.py

### PR DESCRIPTION
**Pull request by taegeonum**
Fix typos in data_adapter.py

### CODE REVIEW

File: keras/engine/data_adapter.py [hunk 1]
```diff
@@ -119,11 +119,11 @@ def get_dataset(self):
 
     Note that the dataset returned does not repeat for epoch, so caller might
     need to create new iterator for the same dataset at the beginning of the
-    epoch. This behavior might change in future.
+    epoch. This behavior might change in the future.
 
     Returns:
       An tf.dataset.Dataset. Caller might use the dataset in different
```
Comment (fchollet) [inline keras/engine/data_adapter.py:125]: There's also one here: 

"""
A `tf.data.Dataset` instance.
"""

Comment (taegeonum) [reply keras/engine/data_adapter.py]: Done. Thanks!

File: keras/engine/data_adapter.py [hunk 2]
```diff
@@ -119,11 +119,11 @@ def get_dataset(self):
 
     Note that the dataset returned does not repeat for epoch, so caller might
     need to create new iterator for the same dataset at the beginning of the
-    epoch. This behavior might change in future.
+    epoch. This behavior might change in the future.
 
     Returns:
-      An tf.dataset.Dataset. Caller might use the dataset in different
-      context, eg iter(dataset) in eager to get the value directly, or in graph
+      A tf.dataset.Dataset. Caller might use the dataset in different
```
Comment (fchollet) [inline keras/engine/data_adapter.py:125]: - It's `tf.data.Dataset` (typo)
- Use backticks

Comment (taegeonum) [reply keras/engine/data_adapter.py]: Done

**Review by fchollet — APPROVED** [review]
LGTM, thanks!

**Pull request merged**